docs/deployment/README.md

```diff
@@ -60,51 +78,13 @@ The process involves:
 
 _If you already are working with an existing `<daac>-deploy` repository that is configured appropriately for the version of Cumulus you intend to deploy or update, skip to [Prepare AWS configuration.](deployment-readme#prepare-aws-configuration)_
 
-Clone template-deploy repo and name appropriately for your DAAC or organization
+Clone template-deploy repo and name appropriately for your DAAC or organization:
 
 ```bash
   $ git clone https://github.com/nasa/template-deploy <daac>-deploy
 ```
 
-Enter repository root directory
-
-```bash
-  $ cd <daac>-deploy
-```
-
-Then run:
-
-```bash
-  $ nvm use
-  $ npm install
-```
-
-If you do not have the correct version of node installed, replace `nvm use` with `nvm install $(cat .nvmrc)` in the above example.
-
-**Note**: The `npm install` command will add the [kes](http://devseed.com/kes/) utility to the `<daac>-deploy`'s `node_modules` directory and will be utilized later for most of the AWS deployment commands.
-
-#### Obtain Cumulus Packages
-
-Cumulus packages are installed from NPM using the `npm install` step above. For information on obtaining additional Cumulus packages, see [Obtaining Cumulus Packages](deployment/obtain_cumulus_packages.md).
-
-### Copy the sample template into your repository
-
-The [`Cumulus`](https://github.com/nasa/cumulus) project contains default configuration values in the `app.example` folder, however these need to be customized for your Cumulus app.
-
-Begin by copying the template directory to your project. You will modify it for your DAAC's specific needs later.
-
-```bash
-  $ cp -r ./node_modules/@cumulus/deployment/app.example ./app
-```
-
-**Optional:** [Create a new repository](https://help.github.com/articles/creating-a-new-repository/) `<daac>-deploy` so that you can track your DAAC's configuration changes:
```

Comment:
This might be worth retaining, depending on how we're expecting users to deploy workflows.   In theory they could import workflow modules from another location instead of including them here, but they're still going to need them in main.tf or another tf file in their deployment package.      Our example in https://github.com/nasa/cumulus-template-deploy/tree/CUMULUS-1556-add-deployment is setup with this in mind.

Reply:
Perhaps change ` so that you can track your DAAC's configuration changes:` to something like `so you can add your workflows and other modules to source control`?

Reply:
f3089cd2a3a424155a2693776abfd86ea00de594